src/Runner/DefaultTestResultCache.php

```diff
@@ -50,7 +50,7 @@ final class DefaultTestResultCache implements \Serializable, TestResultCache
      * $this->defects[$testName] = BaseTestRunner::TEST_SKIPPED;
      * </code>
      *
-     * @var array array<string, int>
+     * @var int[]
      */
     private $defects = [];
```

Comment:
I think this annotation is correct, though

Reply:
Done.